demo/sections/components/RunGraphDemo.vue

```diff
@@ -1,23 +1,39 @@
 <template>
   <p-layout-default class="run-graph-demo">
-    <RunGraph :config="config" :zoom="zoom" class="run-graph-demo__graph" />
-    <p-number-input v-model="zoom" />
+    <RunGraph :config="config" class="run-graph-demo__graph" />
+    <!-- <p-number-input v-model="zoom" /> -->
   </p-layout-default>
 </template>
 
 <!-- eslint-disable camelcase -->
 <script lang="ts" setup>
-  import { endOfHour, startOfHour } from 'date-fns'
-  import { ref } from 'vue'
+  import { randomId } from '@prefecthq/prefect-design'
+  import { endOfHour, startOfHour, startOfDay, endOfDay } from 'date-fns'
+  // import { ref } from 'vue'
   import RunGraph from '@/components/RunGraph.vue'
   import { RunGraphConfig, RunGraphData } from '@/models'
 
-  const zoom = ref(0)
+  // const zoom = ref(0)
```

Comment:
```suggestion
```

Reply:
I'll remove these zoom references in a future PR. For now they're just there so I can test if needed